packages/calcite-components/src/tests/commonTests.ts

```diff
@@ -1323,3 +1323,63 @@ export async function t9n(componentTestSetup: ComponentTestSetup): Promise<void>
     await page.waitForChanges();
   }
 }
+
+/*
+ * @param {ComponentTestSetup} ComponentTestSetup - A component tag, html, or the tag and e2e page for setting up a test.
+ * @param {string} toggleProp - Toggle property to test. Currently, either "open" or "expanded".
+ * @param {string} containerSelector - Container element selector to test for visibility.
+ *
+ * @example
+ * import { emitsOpenCloseTransitionChainedEvents } from "../../tests/commonTests";
+ *
+ * describe("emitsOpenCloseTransitionChainedEvents", () => {
+ *   emitsOpenCloseTransitionChainedEvents("calcite-dropdown", "open", ".container");
+ * });
+ *
+ */
+export function emitsOpenCloseEvents(
```

Comment:
It might be better to avoid implementation specifics in the helper name (namely, `emits` and `events`). This would allow the test helper to cover additional scenarios if needed without having to rename. WDYT about `openClose`? This is going by our current helper naming, 

* `formAssociated`
* `labelable`
* `focusable`
* `reflects`
* `defaults`
* `hidden`
* `accessible`
* `slots`
* `floatingUIOwner`
* `t9n`

Reply:
Sure, what additional scenarios are you thinking about?

Reply:
I don't have specific scenarios in mind, but if another use case comes up, we don't have to worry about renaming the util.